saleor/webhook/transport/utils.py

```diff
@@ -86,6 +88,42 @@ class WebhookResponse:
     duration: float = 0.0
 
 
+class RequestorModelName:
+    # lowercase, as it is returned as such by `model._meta.model_name`
+    APP = "app.app"
+    USER = "account.user"
+
+
+@dataclass
+class DeferredPayloadData:
+    model_name: str
+    model_id: Union[int, UUID]
```

Comment:
All fields are about fetching two different models so they should use common naming. Other options we could use are:
- `instance_name`, `instance_id`, `requestor_instance_name`, `requestor_istance_id`
- `object_name`, `object_id`, `requestor_object_name`, `requestor_object_id` 

I'm not sure if there is much difference TBH, so for now I'll stick to original naming. If anyone thinks otherwise, then feel free to comment and propose a version that works better.

Reply:
`model_name` suggest the `class` name. And this is ok. Renaming `model_name` to `instance_name` will suggest that we are talking about `instance.name`, which is not correct. The dataclass accept `model_name` (class(models.Model) and instance_id of this class. So IMO we can't use values like `instance_name` and `instance_id` together. The model_name is correct IMO, only `model_id` suggests that we are storing ID of the `class(models.Model)`.

Reply:
Fixed to use the form from the comment above.